perf/load/templates/service-graph.gen.yaml

```diff
@@ -582,25 +583,81 @@ spec:
     app: {{ .Values.serviceNamePrefix }}0-4
 status:
 ---
+apiVersion: v1
```

Comment:
can you add a comment about this difference. ?

Reply:
ok added comments. If running with single cluster there is no change, only when running with mutli-clusters it is updated so that it is easy to distribute the workloads across two clusters